spec/ddtrace/integration_spec.rb

```diff
@@ -175,6 +175,31 @@ def agent_receives_span_step3(previous_success)
     end
   end
 
+  describe 'origin tag' do
+    # Sampling priority is enabled by default
+    let(:tracer) { get_test_tracer }
+
+    context 'when #sampling_priority is set on a parent span' do
+      let(:parent_span) { tracer.start_span('parent span') }
+
+      before(:each) do
+        parent_span.tap do
+          parent_span.context.origin = 'synthetics'
+        end.finish
+
+        try_wait_until { tracer.writer.spans(:keep).any? }
+      end
+
+      it do
+        tag_value = parent_span.get_tag(
```

Comment:
Minor, but variable assignments in `it` blocks often belong in `let` or `subject`. This one probably belongs in a `subject` block because the tag is the subject under test. Then this `it` block can become `it { is_expected.to eq('synthetics') }`.